README.md

```diff
@@ -1 +1,870 @@
-# gmelius-package
\ No newline at end of file
+<table>
+    <thead>
+    <tr>
+        <th>Title</th>
+        <th>Last Updated</th>
+        <th>Summary</th>
+    </tr>
+    </thead>
+    <tbody>
+    <tr>
+        <td>Gmelius package</td>
+        <td>September 1, 2023</td>
+        <td>Detailed description of the API of the Gmelius package.</td>
+    </tr>
+    </tbody>
+</table>
+---
+title: Gmelius endpoint
+keywords:
+last_updated: February 02, 2022
+tags: []
+summary: "Detailed description of the API of the Gmelius endpoint."
+sidebar: extensions_sidebar
+permalink: endpoints_gmelius.html
+folder: extensions
+---
+
+# Overview
+
+This endpoint allows direct access to the [Gmelius API](https://developers.gmelius.com/),
+however it provides shortcuts and helpers for most common use cases.
+
+Some features available in this endpoint are:
+
+- Authentication and authorization
+- Direct access to the Gmelius API
+- Helpers for API methods
+
+## Configuration
+
+In order to use the Gmelius you must create an app in the gemlius dashboard. There you need to put the redirect URI that will be shown in the endpoint configuration. Then, you will need to go through an Oauth Authroization process with the button "Connect to Gmelius". This will give the app an authorization code is necessary to get the authorization token.
+
+### Client ID
+
+This value comes from the app created in the Gmelius dashboard.
+
+### Client secret
+
+This value comes from the app created in the Gmelius dashboard.
+
+
+### Code Verifier
+
+String used for doing a PKCE validation when asking for an access token with the authorization code. This field will be automatically field when doing the Oauth connection
+
+### Code Challenge
+
+String used for doing a PKCE validation when asking for an access token with the authorization code. This field will be automatically field when doing the Oauth connection
+
+### Authorization Code
+
+This is used for obtaining the first access token with an authorization code. This field will be automatically field when doing the Oauth connection
+
+### Registered URI
+
+This URL has to be configured in the app created in the Gmelius dashboard.
+
+### Webhook Url
+
+This URL is the one that should be used when creating new webhooks.
+
+### Connect To Gmelius
+
+This button will trigger the Oauth connection process and fill in fields mentioned above.
+
+# Javascript API
```

Comment:
Manually copying the JS API section (without the flows section) helpersDocs.md file has some representation fixes. 

Do not forget to set the variable to access the documentation overview (API_URL_HERE) according to this api.